web/conf/context.jsonld

```diff
@@ -95,6 +95,9 @@
       "@id" : "http://id.loc.gov/ontologies/bibframe/supplement",
       "@container" : "@set"
     },
+    "sublocation" : {
+      "@id" : "http://id.loc.gov/ontologies/bibframe/electronicLocator"
+    },
```

Comment:
just saw that this is the wrong URL. Will push myself a fix.

Reply:
Fixed with https://github.com/hbz/lobid-resources/commit/96031178a0fd07947ce838404a9d37de8c1b2aa0.